ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -534,26 +534,29 @@ public void shouldReturnBadStatementIfStatementFailsValidation() {
   }
 
   @Test
-  public void shouldDistributeCreateStatementEvenIfTopicDoesNotExist() {
+  public void shouldNotDistributeCreateStatementIfTopicDoesNotExist() {
```

Comment:
it's not good to distribute a command we know will fail, but sit there retrying!  We no longer do...